pulp_python/app/models.py

```diff
@@ -41,25 +55,25 @@ class PythonPackageContent(Content):
     packagetype = models.TextField(blank=False)
     name = models.TextField(blank=False)
     version = models.TextField(blank=False)
-    metadata_version = models.TextField(blank=False)
-    summary = models.TextField()
-    description = models.TextField()
-    keywords = ArrayField(models.TextField())
-    home_page = models.TextField()
-    download_url = models.TextField()
-    author = models.TextField()
-    author_email = models.TextField()
-    maintainer = models.TextField()
-    maintainer_email = models.TextField()
-    license = models.TextField()
-    requires_python = models.TextField()
-    project_url = models.TextField()
-    platform = models.TextField()
-    supported_platform = models.TextField()
-    requires_dist = ArrayField(models.TextField())
-    provides_dist = ArrayField(models.TextField())
-    obsoletes_dist = ArrayField(models.TextField())
-    requires_external = ArrayField(models.TextField())
+    metadata_version = models.TextField(null=True)
```

Comment:
I think we should keep the null values if they are set that way in the PyPI metadata, and not override it with a default ""